pkg/rpc/client/client.go

```diff
@@ -118,8 +118,14 @@ func (c *Client) Init() error {
 	if err != nil {
 		return fmt.Errorf("failed to get network magic: %w", err)
 	}
-	c.network = version.Magic
-	c.stateRootInHeader = version.StateRootInHeader
+	c.network = version.Protocol.Network
+	if c.network == 0 {
```

Comment:
It technically can be 0. But `MillisecondsPerBlock` can't (if we can't detect `Protocol` presence in any other way).

Reply:
Fixed.